torch_xla/csrc/aten_xla_bridge.cpp

```diff
@@ -39,15 +39,10 @@ class AtenXlaDeviceMapper {
 
  private:
   AtenXlaDeviceMapper() {
-    if (UseVirtualDevice()) {
-      devices_.emplace_back(ParseDeviceString("SPMD:0"));
-      devices_ordinals_[devices_.back()] = 0;
-    } else {
-      for (auto& device_str :
-           torch_xla::runtime::GetComputationClient()->GetLocalDevices()) {
-        devices_.emplace_back(ParseDeviceString(device_str));
-        devices_ordinals_[devices_.back()] = devices_.size() - 1;
-      }
+    for (auto& device_str :
+         torch_xla::runtime::GetComputationClient()->GetLocalDevices()) {
+      devices_.emplace_back(ParseDeviceString(device_str));
+      devices_ordinals_[devices_.back()] = devices_.size() - 1;
```

Comment:
this will break whole SPMD, I think we need to fix it in a different way...

Reply:
Yes, found UT break.